docs/articles/expensify-classic/expensify-card/Expensify-Card-Perks.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Expensify Card Perks
+description: Get the most out of your Expensify Card with exclusive perks!
+---
+
+
+# Overview
+The Expensify Card is packed with perks, both native to our Card program and through exclusive discounts with partnering solutions. The Expensify Card’s primary perks include: 
+- Access to our premiere Expensify Lounge (with more locations coming soon)
+- Swipe to Win, where every swipe has a chance to win fun personalized gifts for you and your closest friends and family members
+- And unbeatable cash back incentive with each swipe
+Below, we’ll cover all of our exclusive offers in more detail and how to claim discounts with our partners.
+
+# Expensify Card Perks
+
+## Access to the Expensify Lounge
+Our [world-class lounge](https://use.expensify.com/lounge) is now open for Expensify members and guests to enjoy! 
+
+We invite you to visit our sleek San Francisco lounge, where sweeping city views provide the perfect backdrop for a morning coffee to start your day. 
+
+Enjoy complimentary cocktails and snacks in a vibrant atmosphere with blazing-fast WiFi. Whether you want a place to focus on work, socialize with other members, or simply kick back and relax – our lounge is ready and waiting to welcome you.
+
+You can sign up for free [here](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fuse.expensify.com%2F) if you’re not an Expensify member. If you have any questions, reach out to concierge@expensify.com and [check](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fuse.expensify.com%2Flounges) this out for more info.
+
+## Swipe to Win
+Swipe to Win is a new [Expensify Card](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fuse.expensify.com%2Fcard) perk that gives cardholders the chance to send a gift to a friend, family member, or essential worker on the frontlines!
+
+Winners can choose to Send a Smile or Send a Laugh. To start, we’re offering one gift per option:
+
+- **Send A Smile:** Champagne by Expensify
+- **Send a Laugh:** Jenge Set
+
+**How to Participate**
+It’s easy! Once you have an Expensify Card, you just need to start using it. With each swipe, you're automatically entered to win and have a 1 in 250 chance of getting a prize!
+
+**How will I know if I’ve won?**
+Winners will be notified immediately via the Expensify app, and receive additional instructions on how to choose and send their desired gift. 
+
+If you don't have Expensify notifications turned on yet, here are some helpful guides:
+- [Apple Notification Preferences](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fsupport.apple.com%2Fen-us%2FHT201925)
+- [Android Notification Preferences](https://community.expensify.com/home/leaving?allowTrusted=1&target=https%3A%2F%2Fsupport.google.com%2Fandroid%2Fanswer%2F9079661%3Fhl%3Den)
```

Comment:
These links take you to a community page like this:

<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/47436092/33ab5fb6-02ec-4b22-9a24-c01b19f489e2">

Let's just jump straight to the destination:

https://support.google.com/android/answer/9079661?hl=en